samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -132,112 +137,209 @@ public class Assignment
+                        {
+                            requestConfig.Headers.Add(
+                                "Prefer", "include-unknown-enum-members");
+                        });
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-        }
 
-        /// <summary>
-        /// Sets up the assignment resources folder
-        /// </summary>
-        /// <param name="client">Microsoft Graph service client</param>
-        /// <param name="classId">User class id</param>
-        /// <param name="assignmentId">Assignment id</param>
-        /// <returns>EducationAssignment</returns>
-        public static async Task<EducationAssignment> SetupResourcesFolder(
-            GraphServiceClient client,
-            string classId,
-            string assignmentId)
-        {
-            try
+            //Create a draft assignment 
+            public static async Task<EducationAssignment> PatchAsync(
+                GraphServiceClient client,
+                string classId,
+                string assignmentId)
             {
-                return await client.Education
-                    .Classes[classId]
-                    .Assignments[assignmentId]
-                    .SetUpResourcesFolder
-                    .PostAsync();
+                try
+                {
+
+                    var requestBody = new EducationAssignment
+                    {
+                        DisplayName = "Reading and review test updated",
+                    };
+
+                    // Get a Graph client using delegated permissions
+                    return await client.Education
+                        .Classes[classId]
+                        .Assignments[assignmentId]
+                        .PatchAsync(requestBody);
+
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"PatchAsync call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-            catch (Exception ex)
+
+            //Delete the created assignment
+            public static async Task DeleteAsync(
+                GraphServiceClient client,
+                string classId,
+                string assignmentId)
             {
-                throw new GraphException($"SetupResourcesFolder call: {ex.Message}", ex, classId, assignmentId);
+                try
+                {
+                    await client.Education.Classes[classId].Assignments[assignmentId].DeleteAsync();
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"DeleteAsync call: {ex.Message}", ex, classId, assignmentId);
+                }
             }
-        }
 
+            public static async Task<EducationAssignment> SetUpAssignmentFeedbackResourcesFolder(
```

Comment:
add a proper method header

Reply:
Added the header "        /// <summary>
        /// Create Assignment Feedback resources folder
        /// Reference :: https://learn.microsoft.com/en-us/graph/assignments-states-transition
        /// </summary>
        /// <param name="client">Microsoft Graph service client</param>
        /// <param name="classId">User class id</param>
        /// <param name="assignmentId">Assignment id in the class</param>
        /// <returns>EducationAssignment</returns>"